js/api-augment/scripts/pre-gen.cjs

```diff
@@ -6,18 +6,18 @@ const fs = require("fs");
 // eslint-disable-next-line
 const path = require("path");
 
-const interfacePath = path.join(__dirname, '../interfaces');
+const interfacePath = path.join(__dirname, "../interfaces");
```

Comment:
nit: wait, is it `'` or `"`? I just had social-app-template change a bunch to `'` 🤷‍♂️

Reply:
Good catch. I had missed making sure that the prettier config was the same for both e2e and api-augment. Our default is to do single quotes. Updated.